### DISCUSSION TITLE
Feature Request: ...

### DISCUSSION THREAD

**Discussion by includewins0ck2** [duckdb/duckdb#10797] [Ideas]
Now there is only one editor support, DBeaver is good, but many times feel too heavy, I do not know if there is no plan to develop a duckdb proprietary editor, just like sqlite has some classic good use of proprietary editors, such as sqlitestudio, they are very good to use, but also very light.

**Comment by Mause** [comment]
If you are looking for something lighter, I'd recommend https://harlequin.sh/

**Reply by includewins0ck2** [reply]
Thank you very much, the program looks great, but it would be nice if it didn't run at the command line, there are a lot of data analysts out there that aren't necessarily good at using command line programs.
